components/project/experimental/Tabs.tsx

```diff
@@ -11,34 +12,35 @@ export interface ProjectTabsContext {
 }
 
 export interface ProjectTabsProps {
-	default: string,
+	defaultText: string,
```

Comment:
A better name would be `defaultTab`

Reply:
Yes, absolutely, didn't realize what it did until now 🤦‍♂️